tests/func/test_cli.py

```diff
@@ -200,11 +203,21 @@ class A:
         self.assertEqual(parent_dir, current_dir)
 
 
-def test_unknown_command_help(capsys):
+@pytest.mark.skipif(
+    not PY39,
+    reason="exit_on_error for command suggestions requires Python 3.9",
```

Comment:
Could we mock sys.exit(or whatever argparse uses) to still test this?

Reply:
Are we mocking `SystemExit` only for this test? Would the following work?
```python
def test_unknown_command_help(caplog, capsys):
    if not PY39:
        with pytest.raises(SystemExit):
            _ = parse_args(["unknown"])
    try:
        _ = parse_args(["unknown"])
    ...
```

Reply:
We don't need to mock SystemExit, DvcParser doesn't use it.